java/hybridse-proto/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <artifactId>openmldb-parent</artifactId>
         <groupId>com.4paradigm.openmldb</groupId>
-        <version>0.8.6-SNAPSHOT</version>
+        <version>0.9.0</version>
```

Comment:
not mergable since its a release version for java. 
run:
```
./steps/prepare_release.sh 0.9.1.a0 
```
and bump version for next alpha release